packages/itwin/grouping-mapping-widget/src/components/Properties/hooks/useCalculatedPropertiesQuery.ts

```diff
@@ -2,13 +2,13 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import type { IMappingsClient } from "@itwin/insights-client";
+import type { IPropertiesClient } from "@itwin/insights-client";
 import { useQuery } from "@tanstack/react-query";
 import type { GetAccessTokenFn } from "../../context/GroupingApiConfigContext";
 
-export const useCalculatedPropertiesQuery = (iModelId: string, mappingId: string, groupId: string, getAccessToken: GetAccessTokenFn, mappingsClient: IMappingsClient) => {
+export const useCalculatedPropertiesQuery = (iModelId: string, mappingId: string, groupId: string, getAccessToken: GetAccessTokenFn, propertiesClient: IPropertiesClient) => {
```

Comment:
There should only be 1 `usePropertiesQuery` implementation.

Reply:
Done.

Reply:
Rename it to `usePropertiesQuery`

Reply:
Renamed to `usePropertiesQuery`.